### ISSUE TITLE
CRM integration issue - Active Campaign

### ISSUE THREAD

**Issue by anchovy** [verbb/formie#537]
**Description**
Updating a contact's custom fields is wiping their name (firstName, lastName) and phone values if not mapped in the form settings.

See docs: https://developers.activecampaign.com/reference#create-a-contact-new
The only field required is the email field, the firstName, lastName and phone key/value should only be sent if mapped.

**Steps to reproduce**
1. Add a test field to contact in Active Campaign.
2. Add a first, last and phone number to a contact in  Active Campaign.
3. Create a formie form with email & text field and map this to a contacts email and test field. 
4. Make a form submission
5. View the contact in  Active Campaign, you'll see the name and phone number has been reset.

**logic checking a solution**
1. Reset the contact in Active campaign and add their name + phone number.
2. Commenting out the lines 293,294,295 in `vendor/verbb/formie/src/integrations/crm/ActiveCampaign.php` 
3. Make a form submission
5. View the contact in  Active Campaign, you'll see the name and phone number still exist.

**Steps to resolve**
update the code in `vendor/verbb/formie/src/integrations/crm/ActiveCampaign.php` to only send the firstName, lastName, phone values if these have been selected in the Contact Field Mapping for the form and/or have a value.

### Form settings
Please pick from the options below.

- Multi-page form: No
- Submission Method: Ajax
- Client-side Validation: Yes
- Custom Form Templates: Yes

**Additional info**
- Plugin version: 1.4.12
- Craft version: Craft Pro 3.6.10
- Multi-site: No

**Additional context**

**Comment by engram-design** [comment]
Sorry about that. Should be fixed for the next release. To get the fix early, change your `verbb/formie` requirement in `composer.json` to:

```
"require": {
  "verbb/formie": "dev-craft-3 as 1.4.12",
  "...": "..."
}
```

Then run `composer update`.

**Comment by anchovy** [comment]
Sweet / when do you think this'll be released formally? Know from previous pre-releases I've 'forgotten' about this custom require as block and have missed out on subsequent releases!

**Comment by engram-design** [comment]
Likely in 2-3 days!

**Comment by anchovy** [comment]
Monday morning NZ time would be perfect 😃 👍

**Comment by engram-design** [comment]
Fixed in [1.4.13](https://github.com/verbb/formie/releases/tag/1.4.13)
